test/third_party/glbook/Common/esUtil.c

```diff
@@ -53,7 +53,11 @@ EGLBoolean CreateEGLContext ( EGLNativeWindowType hWnd, EGLDisplay* eglDisplay,
    EGLint contextAttribs[] = { EGL_CONTEXT_CLIENT_VERSION, 2, EGL_NONE, EGL_NONE };
 
    // Get Display
+#if __EMSCRIPTEN__
+   display = eglGetDisplay((EGLNativeDisplayType)0);
+#else
    display = eglGetDisplay((EGLNativeDisplayType)x_display);
+#endif

```

Comment:
Is this change specific to wasm64?

And why is it correct - don't we need to give it the proper display that was created?

Reply:
Yes because this casts from a pointer type of an `int`.. which on wasm32 isn't a problem but on wasm64 is a narrowing warning.

Reply:
Hmm... but I guess you are right we could/should use the real value.  It doesn't actually matter here since eglGetDisplay ignores this argument completely, but I think we can come up with a better fix.

Reply:
Done

Reply:
I came up with a better fix which was to fix our definition of EGLNativeDisplayType